### DISCUSSION TITLE
Welcome to QB64pe Discussions!

### DISCUSSION THREAD

**Discussion by SteveMcNeill** [QB64-Phoenix-Edition/QB64pe#269] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Reggie60** [comment]
hi, i'm trying the example here: https://wiki.qb64.dev/qb64wiki/index.php/DLL_Libraries
and i got the DLL file in qb64pe and in system32 and syswow64 too, the program compile and when i start the EXE, i'm getting 
cannot find dynamic library file.

**Reply by a740g** [reply]
PlayMidi32.dll is a 32-bit dynamic link library and will not work with 64-bit versions of QB64-PE. You should try the 32-bit versions of QB64-PE for this example. See https://github.com/QB64-Phoenix-Edition/QB64pe/releases

If MIDI playback is what you are after, then use the built-in MIDI capabilities of QB64-PE. See the following for more information:
https://qb64phoenix.com/qb64wiki/index.php/$UNSTABLE
https://qb64phoenix.com/qb64wiki/index.php/$MIDISOUNDFONT
https://qb64phoenix.com/qb64wiki/index.php/SNDPLAYFILE
https://qb64phoenix.com/qb64wiki/index.php/SNDOPEN
https://qb64phoenix.com/qb64wiki/index.php/SNDPLAY
